### PR TITLE
Update index.html with latest allowOrigins:

### PR DESCRIPTION
**Pull request by johnzheng1975**
Fix issue: https://discuss.istio.io/t/configuring-cors/7458/6

Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/docs/reference/config/networking/virtual-service/index.html [hunk 1]
```diff
@@ -120,8 +120,8 @@ <h2 id="CorsPolicy">CorsPolicy</h2>
         host: ratings.prod.svc.cluster.local
         subset: v1
     corsPolicy:
-      allowOrigin:
```
Comment (wzshiming) [inline content/zh/docs/reference/config/networking/virtual-service/index.html:123]: https://github.com/istio/istio/blob/ff7aac89141d234438cb8da7d3701822d19683b7/pilot/pkg/networking/core/v1alpha3/route/route.go#L786-L800

`AllowOrigin` still works and has no problems

**Comment by ericvn** [comment]
The translation team will have to review this. As this file is pulled from another source (istio.io/api) in the English version, I expect that this change would be to make the Chinese version of this file match the English version.

**Review by SataQiu — COMMENTED** [review]
/ok-to-test

**Comment by kebe7jun** [comment]
@johnzheng1975 你还希望合并这个 PR 嘛？

**Comment by johnzheng1975** [comment]
yes, I think so. @kebe7jun

**Review by kebe7jun — APPROVED** [review]
lgtm

**Pull request merged**